otter/integration/tests/test_scaling.py

```diff
@@ -0,0 +1,114 @@
+"""A quick trial-based test to exercise scale-up and scale-down functionality.
+"""
+
+# TODO(sfalvo): Scale-up functionality isn't finished yet.  I'll finish this
+# after refactoring is done.
+#
+# TODO(sfalvo): Scale-down functionality isn't implemented yet.  I'll finish
+# this after refactoring is done.
+
+
+import json
+import os
+
+from twisted.internet import reactor
+from twisted.internet.task import deferLater
+from twisted.internet.tcp import Client
+from twisted.trial import unittest
+from twisted.web.client import HTTPConnectionPool
+
+from otter import auth
+from otter.integration.lib.identity import IdentityV2
+from otter.integration.lib import autoscale
+from otter.integration.lib.resources import TestResources
+
+
+username = os.environ['AS_USERNAME']
+password = os.environ['AS_PASSWORD']
+endpoint = os.environ['AS_IDENTITY']
+flavor_ref = os.environ['AS_FLAVOR_REF']
+image_ref = os.environ['AS_IMAGE_REF']
+region = os.environ['AS_REGION']
+
+
+def dump_js(js):
+    print json.dumps(js, indent=4)
+
+
+def dump_groups(rcs):
+    for g in rcs.groups:
+        dump_js(g)
+    return rcs
+
+
+def find_end_point(rcs):
+    rcs.token = rcs.access["access"]["token"]["id"]
+    sc = rcs.access["access"]["serviceCatalog"]
+    rcs.endpoints["otter"] = auth.public_endpoint_url(sc, "autoscale", "IAD")
```

Comment:
n/m last comment.  "grin" wasn't finding your example because I was in the wrong directory.  Will fix in next push.